src/test/java/com/ncc/aif/ValidatorPerformanceTest.java

```diff
@@ -0,0 +1,107 @@
+package com.ncc.aif;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import org.apache.commons.io.FileUtils;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.riot.Lang;
+import org.apache.jena.riot.RDFDataMgr;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+
+//TODO: remove me
+public class ValidatorPerformanceTest {
+    private static final String ROOT = "../VERDI/MockTA1Algo/";
+    private static final String SAMPLE = ROOT + "R103.Sample.2019.05.14/";
+    private static final String KB = SAMPLE + "R103.kb.ttl";
+    private static final String SMALL_TA1 = SAMPLE + "IC00120RO.parent.ttl";
+    private static final String MEDIUM_TA1 = SAMPLE + "IC0011WX8.parent.ttl";
+    private static final String BIG_TA1 = ROOT + "R103.2019.05.23/IC0011TIJ.parent.ttl";
+    private static final String GAIA_KB = "/home/HQ/ecurley/Documents/AIDA/2019.05.17-AIDA-795-Threading/big-gaia-sansprivate.ttl";
```

Comment:
You may want to either update the path of this or put a comment above stating that this test requires the `big-gaia-sansprivate.ttl` with a location on where to find it.

Reply:
Good idea. Will comply

Reply:
fixed